Jenkinsfile

```diff
@@ -109,9 +109,7 @@ pipeline {
     }
 
     stage('Push standalone image to DockerHub') {
-      when {
-        branch 'master'
-      }
+      when { tag "v*" }
```

Comment:
Don't we want this to be on a separated PR? it seems like a big change and might be confusing when searching through PR list as the title suggest only a version bump

Reply:
there is a separate commit titled "only publish on tags". I think that takes care of this concern